opencti-platform/opencti-front/src/private/components/events/incidents/Root.tsx

```diff
@@ -209,21 +192,23 @@ class RootIncident extends Component {
                       )}
                     />
                   </Switch>
-                );
-              }
-              return <ErrorNotFound />;
-            }
-            return <Loader />;
-          }}
-        />
+          ) : (
+            <ErrorNotFound />
+          )}
+        </>
       </div>
-    );
-  }
-}
-
-RootIncident.propTypes = {
-  children: PropTypes.node,
-  match: PropTypes.object,
+  );
 };
 
-export default withRouter(RootIncident);
+const RootIncident = () => {
+  const { incidentId } = useParams() as { incidentId: string };
+  const queryRef = useQueryLoading < RootIncidentQuery >(incidentQuery, { id: incidentId });
+  return queryRef ? (
```

Comment:
Do you have any blink on the topbar when you switch right menu in the knowledge page ?

A few days ago, we fix a bug on blinking. You can see the fix in the regions/Root.tsx file for example (the topbar and the StixCoreObjectKnowledgeBar are not in the same place anymore)